torch/serialization.py

```diff
@@ -921,7 +921,8 @@ def load(
         pickle_module: module used for unpickling metadata and objects (has to
             match the :attr:`pickle_module` used to serialize file)
         weights_only: Indicates whether unpickler should be restricted to
-            loading only tensors, primitive types and dictionaries
+            loading only tensors, tensor subclasses, primitive types, dictionaries
+            and any types in :func:`torch.serialization.get_safe_globals`.
```

Comment:
Not sure if pointing to get_safe_globals is sufficient, as main API is `add_safe_globals` isn't it?
```suggestion
            and any types added via  :func:`torch.serialization.add_safe_globals`.
```